apps/files/src/components/FileDetails.vue

```diff
@@ -32,29 +16,30 @@
 import Mixins from '../mixins'
 import OcAppSideBar from 'oc_components/OcAppSideBar.vue'
 import { mapActions, mapGetters } from 'vuex'
+import FileSharingSidebar from './FileSharingSidebar.vue'
 
 export default {
   mixins: [Mixins],
   props: ['items'],
   name: 'FileDetails',
   data () {
     return {
-      drawer: false,
-      tabName: '',
-      active: 0
+      /** String name of the tab that is activated */
+      activeTab: 0
     }
   },
   components: {
-    OcAppSideBar
+    OcAppSideBar,
+    FileSharingSidebar
   },
   methods: {
+    log (index) {
+      alert(index)
+    },
     ...mapActions('Files', ['deleteFiles']),
     close () {
       this.$emit('reset')
     },
-    showSidebar (sideBarName) {
```

Comment:
Please keep this. It is used to highlight one Tab when hitting an Action button in the filelist